src/js/components/Menu/Menu.js

```diff
@@ -205,14 +280,44 @@ class Menu extends Component {
               </Box>
               {align.bottom === 'bottom' ? controlMirror : undefined}
             </ContainerBox>
-          }
-        >
-          {content}
-        </DropButton>
-      </Keyboard>
-    );
-  }
-}
+          </Keyboard>
+        }
+      >
+        {content}
+      </DropButton>
+    </Keyboard>
+  );
+});
+
+Menu.propTypes = {
+  dropAlign: PropTypes.shape({
+    top: PropTypes.string,
+    left: PropTypes.string,
+  }),
+  dropProps: PropTypes.shape({}),
+  items: PropTypes.arrayOf({}),
+  messages: PropTypes.shape({
+    openMenu: PropTypes.string,
+    closeMenu: PropTypes.string,
+  }),
+  justifyContent: PropTypes.string,
+};
+
+Menu.defaultProps = {
+  dropAlign: {
+    top: 'top',
```

Comment:
how and why did we decide on the default values?

Reply:
The default values were already defined in the previous iteration of Menu component. I didn't decide them, but just kept them as they were. Maybe we can discuss the default choices with @ericsoderberghp though!

Reply:
no worries, I probably missed it.